triage/helpers.py

```diff
@@ -0,0 +1,45 @@
+import abc
+
+from api_client import api_client
+
+
+class BaseTriageAnswersManager(abc.ABC):
+    def __init__(self, request):
+        self.request = request
+
+    persist_answers = abc.abstractproperty()
+    retrieve_answers = abc.abstractproperty()
+
+
+class TriageAnswersManager:
+    def __new__(self, request):
+        if request.sso_user is None:
+            return SessionTriageAnswersManager(request)
+        return DatabaseTriageAnswersManager(request)
+
+
+class SessionTriageAnswersManager(BaseTriageAnswersManager):
+    SESSION_KEY = 'TRIAGE_ANSWERS'
+
+    def persist_answers(self, answers):
+        self.request.session[self.SESSION_KEY] = answers
```

Comment:
needs `self.request.session.modified = True`, alternatively we can add `SESSION_SAVE_EVERY_REQUEST = True` to settings

Reply:
I'll do `modified = True` as the latter is less explicit of why it's happening.